contrib/test_jenkins.sh

```diff
@@ -72,6 +77,9 @@ MAKE="make"
 MAKEP="make -j${parallel_jobs}"
 export AUTOMAKE_JOBS=$parallel_jobs
 
+have_ib=$(ibv_devinfo -vv | grep GID || echo "")
+have_ptrace=$(capsh --print | grep Current | grep ptrace || echo "")
```

Comment:
1. wrong grep, it is always false
```
$ capsh --print | grep Current
Current: =
$ capsh --print
Current: =
Bounding set =cap_chown,cap_dac_override,cap_dac_read_search,cap_fowner,cap_fsetid,cap_kill,cap_setgid,cap_setuid,cap_setpcap,cap_linux_immutable,cap_net_bind_service,cap_net_broadcast,cap_net_admin,cap_net_raw,cap_ipc_lock,cap_ipc_owner,cap_sys_module,cap_sys_rawio,cap_sys_chroot,cap_sys_ptrace,cap_sys_pacct,cap_sys_admin,cap_sys_boot,cap_sys_nice,cap_sys_resource,cap_sys_time,cap_sys_tty_config,cap_mknod,cap_lease,cap_audit_write,cap_audit_control,cap_setfcap,cap_mac_override,cap_mac_admin,cap_syslog,35,36
Securebits: 00/0x0/1'b0
 secure-noroot: no (unlocked)
 secure-no-suid-fixup: no (unlocked)
 secure-keep-caps: no (unlocked)

```

2. use `capsh --supports=cap_sys_ptrace`

Reply:
AFAIU, `--supports` is **not** what we need here. It tests whether the kernel version supports a particular capability in general, not whether it is indeed available at the moment.

Indeed, when running it inside docker, I get a positive result for ptrace, which is false.

Reply:
rightm supports is not. maybe grep for "Bounding set ="?